### DISCUSSION TITLE
Support for additional cloud providers (Digital Ocean,...) and maybe even local HA installations (Proxmox)

### DISCUSSION THREAD

**Discussion by leonkosak** [mysticaltech/terraform-hcloud-kube-hetzner#1325] [Ideas]
I know that this project is named "kube-hetzner", but would it be viable to support additional cloud providers (and maybe even Proxmox) that k3s installation would be even more robust, because virtual machines would be running across multiple cloud providers?
One of the challenges would be how to design settings for virtual machine creation across multiple providers.

As far as I can see, kube-hetzner is the most complete, configurable and automated "framework" to setup k3s environment.

Please, share your opinions. :)

**Comment by mysticaltech** [comment]
It's a good idea @leonkosak, quite ambitious, but we will think about it.
